map_org_names.yaml

```diff
@@ -263,7 +263,7 @@ mappings:
   - ['^[[:space:]]*sage[[:space:]]*group[[:space:]]*plc[[:space:]]*$', 'Sage']
   - ['^[[:space:]]*sagem[[:space:]]*$', 'Sagemcom']
   - ['^[[:space:]]*saltstack,?[[:space:]]*inc\\.?[[:space:]]*$', 'SaltStack']
-  - ['^[[:space:]]*(samsung|samsung[[:space:]]+(accelerator|electronics?|sds))[[:space:]]*$', 'Samsung Electronics Co. Ltd.']
+  - ['^[[:space:]]*(samsung[[:space:]]+(accelerator|electronics?|sds))[[:space:]]*$', 'Samsung Electronics Co. Ltd.']
```

Comment:
Not really get it... that change will allow using "Samsung" name but will not change any existing company name, and how is it related to ONAP?

Reply:
My intention was that removing this will not drop the organization name 'Samsung' from the organizations table. Since we need to be able to add organization as just 'Samsung' and my understanding is that the job that checks for org mappings will clean that up and replace it with 'Samsung Electronics Co. Ltd.'

Reply:
We need to update affiliations for a few members of the ONAP community and that update is to change the name of the company affiliations from Samsung Electronics Co. Ltd. to just Samsung

Reply:
![Screen Shot 2020-08-18 at 1 03 05 PM](https://user-images.githubusercontent.com/37811940/90559940-3969ed80-e153-11ea-9d78-6fc24c05cf10.png)

Reply:
I yesterday updated the affiliation for this member to Samsung but the org mapping job updated it to Samsung Electronics Co. Ltd. This is why I wanted to remove the reg ex for just 'Samsung' so that we can add organization as 'Samsung' in addition to Samsung Electronics Co. Ltd.